src/topp/CMakeLists.txt

```
@@ -65,6 +65,7 @@ endif()
 target_link_libraries(FileMerger Boost::regex)
 target_link_libraries(DecoyDatabase Boost::regex)
 target_link_libraries(NucleicAcidSearchEngine Boost::regex)
+target_link_libraries(IsobaricWorkflow Eigen3::Eigen)
```

ah ok so I would need to add it here?
```suggestion
target_link_libraries(IsobaricWorkflow Eigen3::Eigen)
target_link_libraries(MetaProSIP Eigen3::Eigen)
```

---

Yep